judge/models/contest.py

```diff
@@ -54,6 +54,14 @@ class Meta:
 
 
 class Contest(models.Model):
+    SCOREBOARD_VISIBLE = 'V'
+    SCOREBOARD_CONTEST = 'C'
```

Comment:
It's not exactly clear how scoreboards are linked with contests. Should be something like `SCOREBOARD_AFTER_CONTEST`.